apps/cookbook/src/app/showcase/list-load-on-demand-showcase/list-load-on-demand-showcase.component.scss

```
@@ -1,6 +1,8 @@
 @import '../showcase.shared';
 
-fieldset {
-  display: inline-block;
-  margin-bottom: size('s');
+.page-example {
```

Consider naming (I believe this is coming from `Page` example?)
```suggestion
.example-wrapper {
```

---

Actually stole it from `list-showcase.component.html` and `list-showcase.component.scss`. I will not fix those files use of `page-example` as part of this PR as the files are otherwise not related to this PR. 

Anyways should be resolved by: bed618b 😀